lib/charms/layer/zookeeper.py

```diff
@@ -0,0 +1,158 @@
+'''
+zookeeper.py
+'''
+
+from charmhelpers.core import host
+from charms import layer
+from charms.layer.apache_bigtop_base import Bigtop
+from jujubigdata.utils import DistConfig
+from charmhelpers.core.hookenv import (open_port, close_port, log,
+                                       unit_private_ip, local_unit)
+
+def format_node(unit, node_ip):
+    '''
+    Given a juju unit name and an ip address, return a tuple
+    containing an id and formatted ip string suitable for passing to
+    puppet, which will write it out to zoo.cfg.
+
+    '''
+    return (unit.split("/")[1], "{ip}:2888:3888".format(ip=node_ip))
+
```

Comment:
There are a couple of lint errors. Here is how you can fix them. As soon as you have done a "charm build" you go to the output directory (something like ../../trusty/apache-bigtop-zookeeper) and you do a "make lint" the make rule comes from the basic layer (its magic :) )